app/views/shared/_main.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= render Developers::BannerComponent.new([
       Developers::UnseededWarningComponent.new(seedable: Rails.env.development?),
       Developers::InvisibleBannerComponent.new(current_user),
-      Developers::NewFieldsComponent.new(current_user, enabled: Feature.enabled?(:new_developer_fields_banner, user: current_user))
+      Developers::NewFieldsComponent.new(current_user)
```

Comment:
IIRC, we can now remove the parameter and code in `Developers::NewFieldsComponent` to check for "enabled" now, too.

Reply:
Okay i have remove the params for "enabled" and the test for that